src/assets/_project/_blocks/legacy/forms/_forms.scss

```diff
@@ -1260,20 +1253,6 @@ form {
     padding: 0;
     top: inherit;
   }
-  .questions abbr[title*="required"] {
-    position: absolute;
-    width: 0;
-    text-align: center;
-    top: 0.1em;
-    padding: 0;
-    margin: 0;
-    margin-left: -0.5em;
-    border-bottom: none;
-    text-decoration: none;
-    font-size: 1.5em;
```

Comment:
This was put in place for better accessibility reason raised a while ago.
Any reason we are getting rid of it?

Reply:
Which of these for accessibility? We already have a block for asterisk styles .

Multiple styles for the same block is overwriting some behaviour and making asterisk moving weird . Check on this link https://oss-uat.clients.squiz.net/law/legal-mediation-and-justice-of-the-peace/settling-disputes-out-of-court/mediation/apply-for-mediation/_nocache